src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -98,7 +100,7 @@ private static async Task InitializeInstance(ITestOutputHelper output)
 
             var psi = new ProcessStartInfo
             {
-                FileName = "npm",
+                FileName = (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("helix"))) ? "npm" : "yarn",
```

Comment:
Is there a particular reason we use yarn only in Helix environments and not everywhere?

Reply:
You can view this PR as more of a prototype trying to get things working, there were weird issues with using both yarn and npm (one blows away the other), on azdo builds, packages are restored as part of build, but on helix we run it as part of the test job, so this PR is sorta just trying to get things working.  If you have any ideas or things you want to try, feel free to just push directly in to this PR as its really just meant to be a play ground to try and figure out how to get selenium working on helix.

Reply:
Got it -- no worries. I'll see if I have time to poke into this.